Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -1725,6 +1725,8 @@
 
 (define (syscall-unix-in-use-after-launch)
     (syscall-number
+        SYS_ftruncate
+        SYS_fsync
```

Comment:
Is it feasible to perform these file operations in e.g. the Networking process? Long term, I think it would be good to be able to block file operations in the WebContent process.

Reply:
If it is substantial work to avoid these syscalls in the WebContent process, I think it is OK to add them back.

Reply:
> Is it feasible to perform these file operations in e.g. the Networking process? Long term, I think it would be good to be able to block file operations in the WebContent process.

The API is created for performant file access (https://fs.spec.whatwg.org/#api-filesystemfilehandle-createsyncaccesshandle), so the methods are sync (https://fs.spec.whatwg.org/#api-filesystemsyncaccesshandle) and the operations are performed in web process directly. 
Moving to network process means web process needs to be blocked waiting network process's result, which would defeat the goal.